sites/diverseeducation.com/server/routes/index.js

```diff
@@ -1,3 +1,4 @@
+const hiringFeed = require('@cox-matthews-associates/package-global/routes/hiring-feed');
```

Comment:
@Shinsina can you update this to  `const feed = require('@cox-matthews-associates/package-global/routes/feed');`

and  the line 11 below call to  `feed(app, '/hiring-feed');`

then update the imported feeds route line 7 - 10 with the following:
```
module.exports = (app, route = '/feed') => {
  const parseEmbeddedMedia = get(app, 'locals.parseEmbeddedMedia');
  const renderBody = isFn(parseEmbeddedMedia) ? parseEmbeddedMedia : (v) => v;
  app.get(route, asyncRoute(async (req, res) => {
```
then in the future if they want to add new feed routes they should be able to just pass it app && route in theory.